packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -299,6 +300,11 @@ export default {
         getGeoLocationUrl: {
             type: String,
             required: true
+        },
+
+        experiments: {
+            type: Object,
+            required: true
```

Comment:
I think this is fine for now but I'm not sure this needs to be required, we could probably just have it default to an empty object.

Reply:
They were the two things I was torn between. I will update to add a default.